### DISCUSSION TITLE
How to use the operators 'and' or 'or' in Pactum?

### DISCUSSION THREAD

**Discussion by RodrigoOBC** [pactumjs/pactum#326] [Q&A]
Hello, good morning.

I have a scenario where the key 'name' can be null or a string. However, in the documentation, I couldn't find any function that could meet this need.

If possible, I request your help to solve this problem. I'm trying to implement Pactum in my company.

**Comment by ASaiAnudeep** [comment]
We can use [Assert Expressions](https://pactumjs.github.io/api/assertions/expectJsonLike.html#assert-expressions) to implement custom assertions.

```js
expectJsonLike('path.to.property', '$V === null || typeof $V === "string"')
```
